library/kotlin/test/io/envoyproxy/envoymobile/GRPCStreamEmitterTest.kt

```diff
@@ -34,7 +34,7 @@ class GRPCStreamEmitterTest {
       }
 
       override fun close(trailers: Map<String, List<String>>?) {
-        isCloseCalled = true
+        isCloseCalled = trailers == null
```

Comment:
Might want to rename `isCloseCalled` now (see https://github.com/lyft/envoy-mobile/pull/509 for iOS)